RGTichy/CD Ch1 p16 workbook/CD Ch1b.js

```diff
@@ -0,0 +1,3 @@
+for (var i=-52; i <= 1066; i++) {
```

Comment:
Also try to wrap all these lines of code in named functions. Why are you counting from -52 to 1066, I wonder and cannot know without more information.